src/test/java/chessgame/GameTest.java

```diff
@@ -1,19 +1,23 @@
 package chessgame;
 
 
-import chessgame.domain.Command;
-import chessgame.domain.Game;
+import chessgame.domain.*;
 import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 
+import java.util.Map;
+
+import static chessgame.point.PointFixture.*;
+
 class GameTest {
     Game game;
 
     @BeforeEach
     void before() {
-        game = new Game();
+        game = new Game(new Board(ChessBoardFactory.create()),"test");
```

Comment:
```suggestion
        game = new Game(new Board(ChessBoardFactory.create()), "test");
```

lint가 적용 안된 부분이 있네요 😉